moto/moto_api/_internal/moto_random.py

```diff
@@ -0,0 +1,32 @@
+from random import Random
+import string
+import uuid
+
+
+HEX_CHARS = list(range(10)) + ["a", "b", "c", "d", "e", "f"]
+
+
+class MotoRandom(Random):
+    def __init__(self):
+        self._rnd = Random()
```

Comment:
This (and __getattribute__) should not be necessary, as we already extend Random - just call `super()` if necessary